scripts/projectView.js

```diff
@@ -0,0 +1,32 @@
+var projectView={};
+
+//projectView.titleList = function() {
+  //$('article').each(function() {
```

Comment:
Commented-out code should be removed before submitting it. You can refer to old code using commit history, or branch off and remove it and use the new branch for the PR.